docs/guides/tanzu_mission_control_self_managed.md

```diff
@@ -0,0 +1,17 @@
+# Beyond SaaS: Tanzu Mission Control Self-Managed
+
+Tanzu Mission Control Self-Managed, available now, is introducing an air-gapped deployment option, in addition to the currently available SaaS offering, to better support customers in highly regulated industries.
```

Comment:
Would recommend linking to VMware Tanzu Mission Control Self-Managed release note page for text "available now" to https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/1.0/rn/tanzumc-selfmanaged-10-release-notes/index.html

Reply:
I did a re-write of the content of this file to make it a little more readable.

Reply:
Thank you @LandoVMW

Reply:
@LandoVMW - I see that you have added the contents in the md file. In terraform these md files are generated from the corresponding md.tmpl file using a tool. So I have taken the content from your content and added to the md.tmpl file to generate the corresponding md file. So this overrides your commit. I have also kept the NOTE section from the pervious change. Thank you for the refined content. Please take a look at the latest changes.